airgun/entities/organization.py

```diff
@@ -101,3 +101,12 @@ def step(self, *args, **kwargs):
         if not org_name:
             raise ValueError('Specify proper value for org_name parameter')
         self.view.taxonomies.select_org(org_name)
+
+    def post_navigate(self, _tries, *args, **kwargs):
+        """Handle alert screen if it's present"""
+        wrong_context_view = WrongContextAlert(self.view.browser)
+        if wrong_context_view.is_displayed:
+            wrong_context_view.back.click()
+            self.view.browser.wait_for_element(
+                self.view.menu.__locator__(), exception=False, ensure_page_safe=True)
```

Comment:
Please double check if we still need to pass locator instead of widget here with new `wait_for_element()` implementation.

Reply:
yeap
```
return self.elements(locator,
                                     parent=parent,
                                     check_visibility=visible,
                                     check_safe=ensure_page_safe)
```

Reply:
Take a look at `browser.elements`, it can work with locator, widget or webelement. So it will definitely accept widget.

Reply:
Previously wait_for_element wasn't even trying to wait for widget, only for locator. But since the logic was updated i hope it can work with widgets correctly. If not - we'll have to update all our wait_for_element calls too.